<!--
  1 failure:  :disappointed_rel...
  1 warning:  There is no linke...
  1 messages
  
-->

<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  :disappointed_relieved: `npm test` failed. See below for details.
  </td>
    </tr>
  </tbody>
</table>


<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  There is no linked JIRA ticket in the PR body. Please include the URL of the relevant JIRA ticket. If you need to, you may file a ticket on <a href="https://jira.appcelerator.org/secure/CreateIssue!default.jspa">JIRA</a>
  </td>
    </tr>
  </tbody>
</table>


<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Messages</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:book:</td>
      <td>

  ```
> titanium-mobile@7.2.0 test /Users/build/jenkins/workspace/dk_titanium_mobile_PR-10020-AX7L5XS7QOPHUR2V3X4BYLQZORTURST2TJTWB45YBY4MZLUCJ45Q
> grunt

Running "appcJs:src:lintOnly" (appcJs) task

Running "eslint:src" (eslint) task

Running "checkFormat:ios" (checkFormat) task

Running "checkFormat:android" (checkFormat) task

Running "validate:docs" task
Warning: Docs validation failed with exit code: 1
Output: [ERROR] ./Titanium/UI/ButtonBar.yml
Titanium.UI.ButtonBar: found 1 error(s)!
	properties
		style
			constants
				Invalid constant: Titanium.UI.iOS.SystemButtonStyle.*

[ERROR] ./Titanium/UI/iOS/TabbedBar.yml
Titanium.UI.iOS.TabbedBar: found 1 error(s)!
	properties
		style
			constants
				Invalid constant: Titanium.UI.iOS.SystemButtonStyle.*

[ERROR] Found 2 error(s)!
[ERROR] The following files have YAML syntax errors: 
[ERROR] ./Titanium/Android/QuickSettingsService.yml
YAMLException: duplicated mapping key at line 85, column 5:
        name: isLocked
        ^
[ERROR] ./Titanium/UI/DashboardView.yml
YAMLException: duplicated mapping key at line 71, column -43:
        deprecated:
        ^
[ERROR] ./Titanium/UI/iOS/SystemButtonStyle.yml
YAMLException: duplicated mapping key at line 49, column -14:
        description: |
        ^
[ERROR] ./Titanium/UI/iPhone/ActivityIndicatorStyle.yml
YAMLException: duplicated mapping key at line 12, column -91:
    deprecated:
    ^
[ERROR] ./Titanium/UI/iPhone/SystemButtonStyle.yml
YAMLException: duplicated mapping key at line 68, column -14:
        description: |
        ^
 Use --force to continue.

Aborted due to warnings.
npm ERR! Test failed.  See above for more details.

```
  </td>
    </tr>
  </tbody>
</table>


<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

--- a/apidoc/Titanium/Android/QuickSettingsService.yml
+++ b/apidoc/Titanium/Android/QuickSettingsService.yml
@@ -82,12 +82,12 @@ methods:
     returns:
         type: String
 
-    name: isLocked
+  - name: isLocked
     summary: Returns 'true' if the device is currently locked, 'false' otherwise.
     returns:
         type: Boolean
 
-    name: isSecure
+  - name: isSecure
     summary: Returns 'true' if the device is in secure state, 'false' otherwise.
     returns:
         type: Boolean

--- a/apidoc/Titanium/UI/DashboardView.yml
+++ b/apidoc/Titanium/UI/DashboardView.yml
@@ -68,8 +68,6 @@ events:
       - name: item
         summary: Item that was dragged.
         type: Titanium.UI.DashboardItem
-    deprecated:
-        since: "3.0.0"
         
   - name: dragStart
     deprecated:
@@ -80,8 +78,6 @@ events:
       - name: item
         summary: Item that was dragged.
         type: Titanium.UI.DashboardItem
-    deprecated:
-        since: "3.0.0"
         
   - name: dragend
     summary: Fired when an item finishes being dragged in edit mode.

--- a/apidoc/Titanium/UI/iOS/SystemButtonStyle.yml
+++ b/apidoc/Titanium/UI/iOS/SystemButtonStyle.yml
@@ -46,7 +46,6 @@ properties:
         
         When used with [ButtonBar](Titanium.UI.ButtonBar) or [TabbedBar](Titanium.UI.iOS.TabbedBar), 
         specifies a standard bar with a heavier border.
-    description: |
     type: Number
     permission: read-only
     

--- a/apidoc/Titanium/UI/iPhone/ActivityIndicatorStyle.yml
+++ b/apidoc/Titanium/UI/iPhone/ActivityIndicatorStyle.yml
@@ -9,9 +9,6 @@ extends: Titanium.Proxy
 since: "0.9"
 platforms: [iphone, ipad]
 createable: false
-deprecated:
-    since: "5.1.0"
-    notes: Use <Titanium.UI.ActivityIndicatorStyle> instead.
 properties:
   - name: BIG
     summary: Large white spinning indicator.

--- a/apidoc/Titanium/UI/iPhone/SystemButtonStyle.yml
+++ b/apidoc/Titanium/UI/iPhone/SystemButtonStyle.yml
@@ -65,7 +65,6 @@ properties:
         
         When used with [ButtonBar](Titanium.UI.ButtonBar) or [TabbedBar](Titanium.UI.iOS.TabbedBar), 
         specifies a standard bar with a heavier border.
-    description: |
     type: Number
     permission: read-only
     

--- a/package.json
+++ b/package.json
@@ -62,7 +62,7 @@
     "grunt-cli": "^1.2.0",
     "grunt-contrib-clean": "^1.0.0",
     "grunt-mocha-test": "^0.13.2",
-    "js-yaml": "~3.2.2",
+    "js-yaml": "^3.11.0",
     "mocha": "*",
     "optimist": "^0.6.1",
     "pagedown": "~1.1.0",

